src/common/types/store-data/needs-review-card-selection-store-data.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+export type NeedsReviewCardSelectionData = { [resultInstanceUid: string]: boolean };
+
+export interface RuleExpandCollapseData {
+    isExpanded: boolean;
+    cards: NeedsReviewCardSelectionData;
+}
+
+export interface RuleExpandCollapseDataDictionary {
+    [ruleId: string]: RuleExpandCollapseData;
+}
+
+export interface NeedsReviewCardSelectionStoreData {
```

Comment:
What would your thoughts be on sharing some of these types/interfaces with the other CardSelectionStore? `RuleExpandCollapseDataDictionary` in particular doesn't feel like it needs to be duplicated, but I could see an argument for sharing the rest as a standard set of card selection store related types.

Reply:
This is a good point, I've changed it to use a type that simply points to the existing data types. There's no reason we can't re-use them currently, and this makes it fairly easy to decouple down the road if we ever decide we need to! Good catch!